test/test_transforms.py

```diff
@@ -230,10 +230,10 @@ def _test_librosa_consistency_helper(n_fft, hop_length, power, n_mels, n_mfcc, s
             librosa_mel = librosa.feature.melspectrogram(y=sound_librosa, sr=sample_rate,
                                                          n_fft=n_fft, hop_length=hop_length, n_mels=n_mels,
                                                          htk=True, norm=None)
+            librosa_mel_tensor = torch.from_numpy(librosa_mel)
             torch_mel = melspect_transform(sound).squeeze().cpu().t()
 
-            # lower tolerance, think it's double vs. float
```

Comment:
why did you remove the comment?

Reply:
it might be a double or float depending on what the function returns/default dtype

Reply:
The function being "librosa"? If that matters, please check it and branch or assert.